lib/bike_brigade/delivery.ex

```diff
@@ -677,10 +677,21 @@ defmodule BikeBrigade.Delivery do
       iex> list_items()
       [%Item{}, ...]
 
+      iex> list_items(42)
+      [%Item{program_id: 42, …}, …]
+
   """
-  def list_items do
+  def list_items(program_id \\ nil) do
     query = from i in Item, order_by: i.name
 
+    query =
+      if program_id do
+        query
+        |> where([i], i.program_id == ^program_id)
+      else
+        query
+      end
```

Comment:
Do we prefer to filter down by program id here, or rather leave it up to the caller, similar to [how campaigns are filtered below](https://github.com/bikebrigade/dispatch/pull/125/files#diff-c47a1aff9bf95e01794823e9077bbcf6c1b6321beadf2ace8ee9496415766c88R58)?